include/gz/physics/config.hh.in

```diff
@@ -1,16 +1,35 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
 /* Config.hh. Generated by CMake for @PROJECT_NAME_NO_VERSION@. */
 
-#ifndef IGNITION_${IGN_DESIGNATION_UPPER}_CONFIG_HH_
-#define IGNITION_${IGN_DESIGNATION_UPPER}_CONFIG_HH_
+#ifndef GZ_PHYSICS_CONFIG_HH_
+#define GZ_PHYSICS_CONFIG_HH_
 
 /* Version number */
-#define IGNITION_${IGN_DESIGNATION_UPPER}_MAJOR_VERSION ${PROJECT_VERSION_MAJOR}
-#define IGNITION_${IGN_DESIGNATION_UPPER}_MINOR_VERSION ${PROJECT_VERSION_MINOR}
-#define IGNITION_${IGN_DESIGNATION_UPPER}_PATCH_VERSION ${PROJECT_VERSION_PATCH}
+#define GZ_PHYSICS_MAJOR_VERSION ${PROJECT_VERSION_MAJOR}
+#define GZ_PHYSICS_MINOR_VERSION ${PROJECT_VERSION_MINOR}
+#define GZ_PHYSICS_PATCH_VERSION ${PROJECT_VERSION_PATCH}
+
+#define GZ_PHYSICS_VERSION "${PROJECT_VERSION}"
+#define GZ_PHYSICS_VERSION_FULL "${PROJECT_VERSION_FULL}"
 
-#define IGNITION_${IGN_DESIGNATION_UPPER}_VERSION "${PROJECT_VERSION}"
-#define IGNITION_${IGN_DESIGNATION_UPPER}_VERSION_FULL "${PROJECT_VERSION_FULL}"
+#define GZ_PHYSICS_ENGINE_INSTALL_DIR "${IGNITION_PHYSICS_ENGINE_INSTALL_DIR}"
 
-#define IGNITION_PHYSICS_ENGINE_INSTALL_DIR "${IGNITION_PHYSICS_ENGINE_INSTALL_DIR}"
+#define GZ_PHYSICS_VERSION_HEADER "Gz Physics, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2022 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
```

Comment:
Interesting that it didn't have a header yet, I guess it's because it doesn't have a CLI. It doesn't hurt to add it, I'd just spell out Gazebo and put the year as 2017

```suggestion
#define GZ_PHYSICS_VERSION_HEADER "Gazebo Physics, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2017 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
```